### PR TITLE
telnet: improve descriptions

### PR DESCRIPTION
**Pull request by waldyrious**
also format "x" as a token in the last example.

Note: I'm not entirely sure of the last example's description. Confirmations of its correctness welcome :)

### CODE REVIEW

File: pages/common/telnet.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
-- To exit a telnet session:
+- Exit a telnet session:
 
 `quit`
 
-- Default escape character:
+- Emit the default escape character:
 
 `CTRL + ]`
```
Comment (agnivade) [inline pages/common/telnet.md]: Shouldn't this be 'CTRL+]' according to your new convention ?

File: pages/common/telnet.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
 # telnet
 
-> Telnet is used to connect to a specified port of a host.
+> Connect to a specified port of a host using the telnet protocol.
 
 - Telnet to a certain port:
```
Comment (agnivade) [inline pages/common/telnet.md]: While we are on this page, maybe we should start with a simple 'telnet {{host}}' example and say that telnet connects to port 23 by default.

File: pages/common/telnet.md (hunk 1 above)
Comment (waldyrious) [inline pages/common/telnet.md]: It should, but that makes it hard to read. Shall we adopt a convention of adding spaces around the + sign? That sounds reasonable and in line with the project's ethos (prioritizing readability over compactness).

Comment (agnivade) [inline pages/common/telnet.md]: sure :+1:

Comment (waldyrious) [inline pages/common/telnet.md]: Done in #1356.

Comment (sbrl) [inline pages/common/telnet.md]: As a user, I might not know what an 'escape character' does. Perhaps it should be made more clear that terminates the session?

Comment (waldyrious) [inline pages/common/telnet.md]: "session escape character"? "session termination character"? In fact, I'm not sure it's intruitive to call this a character. Maybe "character combination", "keyboard shortcut", or something to that effect?

File: pages/common/telnet.md (hunk 2 above)
Comment (waldyrious) [inline pages/common/telnet.md]: Not sure how I missed this comment -- probably the notification linked directly to the second comment you made. Yes, agree, and will make this change as soon as we reach consensus in the second thread below.

File: pages/common/telnet.md (hunk 1 above)
Comment (sbrl) [inline pages/common/telnet.md]: Yeah, that would work I think

Comment (waldyrious) [inline pages/common/telnet.md]: Which one? xD

Comment (jeeftor) [inline pages/common/telnet.md]: Is this ready for review?

Comment (waldyrious) [inline pages/common/telnet.md]: I'm waiting for @sbrl to reply :)

Comment (sbrl) [inline pages/common/telnet.md]: Ahhh I see
My comment doesn't specify which one I'd prefer. I vote for character combination / keyboard shortcut

**Comment by sbrl** [comment]
Merge complete!

**Pull request merged**